### DISCUSSION TITLE
Unity 프로젝트의 Web 배포에 대하여.

### DISCUSSION THREAD

**Discussion by hongsam14** [hongsam14/myCPU#28] [Q&A]
친구가 Unity로 웹게임을 만들 수 있냐고 물어봤습니다. 제가 알기론 WebGL을 사용하여 Web에서 구동은 가능한 걸로 알고 있습니다만, 실제 서비스에서의 배포는 또 다른 문제인 것 같아 궁금하였습니다. 특히 모바일 웹은 지원 안하는 것으로 알고 있는데, 실제로 Unity 프로젝트를 웹 브라우저에서 구동이 되게 만들기도 하나요? 그리고 웹게임을 만드는데 Unity가 적합할까요?

**Comment by jongfeel** [comment]
이건 바로 답변 드리는게 좋겠네요

- 특히 모바일 웹은 지원 안하는 것으로 알고 있는데
  - Unity WebGL mobile web 지원 합니다. 왜냐하면 HTML5 기반의 web assembly로 배포되기 때문이죠
- 실제로 Unity 프로젝트를 웹 브라우저에서 구동이 되게 만들기도 하나요?
  - 가끔 크로스플랫폼 서비스 할때 하긴 합니다.
  - 그리고 딱 한번 web으로 배포한 unity 시뮬레이션 프로젝트를 보긴 했습니다.
  - 그런데 일반적이지는 않은 건 맞아요.
- 웹게임을 만드는데 Unity가 적합할까요?
  - 웹게임이면 가벼운 캐주얼 게임일 확률이 높은데, 그런건 순수 웹 기술로 만드는게 좋습니다.
  - Unity는 괜찮은 하드웨어에서 괜찮은 성능의 렌더링을 보여주는 게임을 개발하는데 적합합니다.
